File: ProgII/src/Model/Marca.java
package Model;

public class Marca{
	
	public static final String MARCA_ID = "idMarca";
    public static final String MARCA_DESC = "descricao";
	
	private int idMarca;
	private String descricaoMarca;
	
	public int getIdMarca() {
		return idMarca;
	}
	
	public void setIdMarca(int idMarca) {
		this.idMarca = idMarca;
	}
	
	public String getDescricaoMarca() {
		return descricaoMarca;
	}
	
	public void setDescricaoMarca(String descricaoMarca) {
		this.descricaoMarca = descricaoMarca;
	}

	@Override
	public String toString() {
		return descricaoMarca;
	}
	
	
	
}

File: Sistema_Web/models/Marca.php
<?php


class Marca{
    private $idMarca;
    private $descricao;

    /**
     * @return mixed
     */
    public function getIdMarca()
    {
        return $this->idMarca;
    }

    /**
     * @param mixed $idMarca
     */
    public function setIdMarca($idMarca)
    {
        $this->idMarca = $idMarca;
    }

    /**
     * @return mixed
     */
    public function getDescricao()
    {
        return $this->descricao;
    }

    /**
     * @param mixed $descricao
     */
    public function setDescricao($descricao)
    {
        $this->descricao = $descricao;
    }



}